src/shared/src/types/project-types.ts

```diff
@@ -71,6 +71,7 @@ export interface DescriptionBullet {
   dateAdded: Date;
   dateDeleted?: Date;
   userChecked?: User;
+  dateChecked?: Date;
```

Comment:
we didnt have this before??

Reply:
No it was just wrong 💀, it was using dateAdded instead of dateChecked